addons/html_builder/static/src/builder/snippets_menu.js

```diff
@@ -96,6 +88,8 @@ export class SnippetsMenu extends Component {
                         this.state.currentOptionsContainers = currentOptionsContainers;
                         this.setTab("customize");
                     },
+                    unsplittable_node_predicates: (node) =>
+                        node.querySelector("[data-oe-translation-source-sha]"),
```

Comment:
Maybe just add it in translation mode ? (Or add a comment) We maybe want use another config in translation ?

Reply:
remove few plugins etc ?